app.js

```diff
@@ -1,13 +1,17 @@
 const express = require("express");
+const { getArticleById } = require("./controllers/articles");
 const { getTopics } = require("./controllers/topics");
 const app = express();
 
 app.get("/api/topics", getTopics);
+app.get("/api/articles/:article_id", getArticleById);
 
 // Handling JavaScript errors
 app.use((error, req, res, next) => {
 	console.log(error.message);
+	console.log(error.status);
```

Comment:
Make sure to move console logs. The only console log for errors should really be with a 500 error

Reply:
sure, will do ! thanks